Assets/Scripts/Gameplay/GameState/BossRoomState.cs

```diff
@@ -16,14 +16,11 @@ namespace Unity.Multiplayer.Samples.BossRoom.Server
     /// Server specialization of core BossRoom game logic.
```

Comment:
I see the simplification of BossRoomState, but not of CharSelectState? Did I miss it?

Reply:
CharSelectState family is rather complex on both client and server sides - definitely not simple bridges.